cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -100,12 +100,17 @@ processors:
     ballast_size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
     check_interval: 2s
     limit_mib: ${SPLUNK_MEMORY_LIMIT_MIB}
-  ## detect if the collector is running on a cloud system
-  ## important for creating unique cloud provider dimensions
+  # detect if the collector is running on a cloud system
+  # important for creating unique cloud provider dimensions
   resourcedetection:
     detectors: [system, gce, ecs, ec2, azure]
     override: false
 
+  # Same as above but overrides resource attributes set by receivers
+  resourcedetection/internal:
+    detectors: [system, gce, ecs, ec2, azure]
+    override: true
```

Comment:
Actually is there a reason to create another receiver ? we already have one with the same config

Reply:
ok, nvm, it's not the config :)